src/librbd/internal.cc

```diff
@@ -869,7 +869,13 @@ bool compare_by_name(const child_info_t& c1, const child_info_t& c2)
     }
 
     if (old_format) {
-      r = create_v1(io_ctx, image_name.c_str(), size, order);
+      if ( !getenv("RBD_FORCE_ALLOW_V1") ) {
+        lderr(cct) << "Format 1 image creation unsupported. " << dendl;
+        return -EINVAL;
+      } else {
```

Comment:
Nit: you can drop the ```else``` clause since the previous clause will return.

Reply:
Okay I'll have a look, thanks for pointing this out.